src/peers/PeersTable/PeersTable.js

```diff
@@ -37,7 +37,7 @@ export class PeersTable extends React.Component {
   }
 
   locationCellRenderer = ({ rowData }) => (
-    <span>
+    <span title={ rowData.location ? rowData.location : this.props.t('unknownLocation')}>
```

Comment:
I think a shorter notation should also work? (but check first):
```suggestion
    <span title={ rowData.location || this.props.t('unknownLocation')}>
```

Reply:
It does!